src/rgw/rgw_rest.cc

```diff
@@ -790,6 +790,19 @@ int RGWGetObj_ObjStore::get_params()
     mod_pg_ver = s->info.env->get_int("HTTP_DEST_PG_VER", 0);
   }
 
+  /* start gettorrent */
+  const char *query_str = s->info.env->get("QUERY_STRING");
+  if (0 == strcmp(query_str, GET_TORRENT))
```

Comment:
there may be other params in the query string, right? i think you want to use `s->info.args.exists(GET_TORRENT)` here instead